### DISCUSSION TITLE
Warning. Your credentials have been leaked by a user named Sex444. Please perform marisex to get new credentials.

### DISCUSSION THREAD

**Discussion by Nrmot** [Nrmot-GithubRepo/Project-CBT#33] [Project CBT: Latest Dev Releases]
SIKE! ITS ANOTHER PROJECT CBT SNAPSHOT!!!! 

nothing changed. 

download the new snapshot and there's nothing changed. 

~~im such a fucking genius for totally uploading a new snapshot with no fucking effort~~

<hr /><em>This discussion was created from the release <a href='https://github.com/Nrmot-GithubRepo/Project-CBT/releases/tag/Alpha-0.0.3-snapshot2'>Warning. Your credentials have been leaked by a user named Sex444. Please perform marisex to get new credentials.</a>.</em>

**Comment by Higuys153** [comment]
Fuck you
